appcenter/ios/appcenter-core.podspec

```diff
@@ -3,7 +3,7 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, '../package.json')))
 
 Pod::Spec.new do |s|
-  s.name              = package['name']
+  s.name              = 'appcenter-core'
```

Comment:
Let's add a short description why we use a string literal instead of the variable.

Reply:
Bump